pkg/operator/starter.go

```diff
@@ -177,7 +178,14 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 		operatorClient,
 		versionRecorder,
 		controllerContext.EventRecorder,
-	)
+	).WithDegradedInertia(status.MustNewInertia(
+		2*time.Minute,
+		status.InertiaCondition{
+			ConditionTypeMatcher: regexp.MustCompile("^EtcdMembersDegraded$"),
```

Comment:
Side note: If library-go folks are onboard ideally, the `InertiaCondition` should have something like `ExactyTypeMatcher`, as we here want the exact match anyways and would say us not needing to call regex. Maybe for a future improvement, not blocking at all.

Reply:
I agree. It makes code more readable and easier to maintain. But supporting regex to cover multiple conditions such as ("^(EtcdMembers|EtcdCertSigner|ClusterMember)ControllerDegraded$") is useful too.